# scGEN - Gene-aware Embedded Network via Hard-sample Mining for Single-cell RNA-seq Data

Recent advances in transcriptional sequencing have greatly improved our ability to explore heterogeneity within cell populations, with unsupervised clustering playing a crucial role in identifying these differences. The quality of clustering significantly influences downstream analyses. However, many current methods fail to adequately address challenging cell populations, particularly those at the boundaries between different cell states or developmental stages. This limitation risks the loss of essential biological insights. Attempts to apply hard-sample mining techniques from traditional machine learning to single-cell analysis have fallen short in capturing the complexity of cellular relationships, often missing key biological context. Furthermore, current methods typically rely on highly variable genes (HVGs) without considering differences in their expression levels. HVGs with lower expression may still carry necessary disease signals, and disregarding these subtle cues limits the model's ability to capture clinically relevant information. To overcome these limitations, we present a single-cell gene-aware embedded network (scGEN), a gene-aware embedding network designed to capture the topological relationships among cells while emphasizing challenging samples to enhance clustering accuracy. scGEN employs an adaptive weighting strategy to highlight the most informative and challenging cells. It introduces two fine-tuning parameters to prioritize HVGs based on their expression levels, allowing the model to detect nuanced, low-expression signals crucial for understanding complex biological processes. Validation on eight independent datasets demonstrates that scGEN consistently outperforms seven other leading clustering approaches, underscoring its effectiveness in revealing significant biological structures.
